src/main/target/MATEKF411/target.h

```diff
@@ -30,6 +30,9 @@
 #define BEEPER_PIN              PB2
 #define BEEPER_INVERTED
 
+#undef  ENABLE_DSHOT_DMAR
```

Comment:
Can you please remove this - this will probably break all kinds of configurations that work now.